ansible/roles/jenkins-master/defaults/main.yaml

```diff
@@ -36,3 +36,5 @@ jenkins_plugins:
   - timestamper
```

Comment:
Regarding the commit msg, yes, we could do that. I prefer as you implemented, I'd just like to make it clearer in commit msg why we are doing this instead of putting the credentials in the system where the script would run (more obvious solution).